modules/interactive_music/audio_stream_playlist.cpp

```diff
@@ -390,6 +411,41 @@ bool AudioStreamPlaybackPlaylist::is_playing() const {
 	return active;
 }
 
+void AudioStreamPlaybackPlaylist::change_clip(int stream_index) {
+	ERR_FAIL_COND(stream_index < 0 || stream_index >= playlist->stream_count);
```

Comment:
You can use ERR_FAIL_INDEX for a better error message here.

Reply:
Didn't know that, thanks. Done.